example/example_training_loops.py

```diff
@@ -109,14 +110,21 @@ def main():
 
     # Train the policy or value based approach
     if args["algorithm"] == "PPO":
-        ppe.ppo_train(env, agent, args)
-        ppe.evaluate_ppo_network(env, agent, args)
+        #create the record class
+        record = Record(networks={'actor':agent.actor_net, 'critic': agent.critic_net}, config={})
```

Comment:
It's useful here to add the `args` dictionary as the config. It means that with every training run, we know the exact hyperparameters used to produce the results.

Also adding a `checkpoint_freq` is useful. Most of the time, we do not do complete training runs, and so having checkpoints mean that all training isn't lost.

Reply:
thank you, i have resolved this in commit 18d75fd